app.py

```diff
@@ -82,25 +82,45 @@ def get_criterion(id_):
 	except Exception as e:
 		return(str(e))
 
-@app.route("/links")
+@app.route("/links", methods=["GET"])
 def get_links():
-	try:
-		links=Link.query.all()
-		return  jsonify([link.serialize() for link in links])
-	except Exception as e:
-		return(str(e))
+	links=Link.query.all()
+	return  jsonify([link.serialize() for link in links])
+
+@app.route("/links", methods=["POST"])
+@cross_origin(headers=["Content-Type", "Authorization"])
+@requires_auth
+def create_link():
+  data = request.form
```

Comment:
@brittanylau pointed out that this should be `request.get_json()` -- I'll fix this issue everywhere it appears in a follow-up PR.